src/rest.cpp

```diff
@@ -610,7 +610,7 @@ static bool rest_blockhash_by_height(const util::Ref& context, HTTPRequest* req,
     std::string height_str;
     const RetFormat rf = ParseDataFormat(height_str, str_uri_part);
 
-    int32_t blockheight;
+    int32_t blockheight = 0;
```

Comment:
Heh, nice find. Maybe split this up as a separate pull request, since it seems unrelated to the changes here?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18740#discussion_r414772851

> Maybe split this up as a separate pull request, since it seems unrelated to the changes here?

Yes it should be basically be unrelated. I created https://github.com/bitcoin/bitcoin/pull/18785, but am also keeping the commit here for now, since it does seem to be needed to avoid travis failing with the other changes here https://travis-ci.org/github/bitcoin/bitcoin/jobs/678453061#L7157